src/common/tracer.h

```diff
@@ -0,0 +1,66 @@
+#ifndef TRACER_H_
+#define TRACER_H_
+
+#define SIGNED_RIGHT_SHIFT_IS 1
+#define ARITHMETIC_RIGHT_SHIFT 1
+
+#include <iostream>
+#include <arpa/inet.h>
+#include <yaml-cpp/yaml.h>
+
+#include <jaegertracing/Tracer.h>
+
+#ifndef WITH_JAEGER
+  #define WITH_JAEGER
+#endif
+
+typedef std::unique_ptr<opentracing::Span> Span;
+
+class Jager_Tracer{
```

Comment:
* please fix indentation
* please avoid one line "if" without {}
e.g. this is causing a bug in line 27

Reply:
noted